src/micro/extends.html

```diff
@@ -49,10 +49,14 @@
 
     _getExtendedPrototype: function(tag) {
       var isNative = tag && tag.indexOf('-') < 0;
-      var proto = isNative ? 
+      var proto = isNative ?
         this._getExtendedNativePrototype(tag) :
         this._getExtendedElementPrototype(tag);
-      proto.__extendsNativeElement = isNative;
+      // let `__extendsNativeElement` walk prototype chain to allow
+      // extensions of extensions of native elements
+      if (isNative) {
```

Comment:
If you set `__extendsNativeElement` on every prototype, then you can't extend elements that extend native elements to do the right call to `createElement`

```js
class XInput extends Polymer.Extend('input'){}
XInput.prototype.__extendsNativeElement === true;
// new XInput() -> <input is="x-input">
class XExtendInput extends XInput {}
XExtendInput.prototype.__extendsNativeElement === false;
// new XExtendInput() -> <x-extend-input>
```